internal/helm/repository/oci_chart_repository_test.go

```diff
@@ -46,8 +47,12 @@ type mockRegistryClient struct {
 	LastCalledURL string
 }
 
-func (m *mockRegistryClient) Tags(url string) ([]string, error) {
-	m.LastCalledURL = url
+func (m *mockRegistryClient) Tags(urlStr string) ([]string, error) {
+	// there was a trailing slash in the repository url string
+	if strings.Contains(urlStr, "//") {
+		return nil, fmt.Errorf("remote reference: invalid_reference: invalid repository")
+	}
```

Comment:
I don't think we need to do this. It's a minimal mock of a registry client and such checks are usually not needed in mocks where we just try to mock real scenario.
The actual validation of the URL we are trying to achieve in the test for this PR can be done by just comparing the URL from `Get()` and the expected URL, which is already being done in the test below.